### DISCUSSION TITLE
Welcome to AzUrlShortener Discussions!

### DISCUSSION THREAD

**Discussion by fboucher** [microsoft/AzUrlShortener#234] [General]
## 👋 Welcome! ☁️ 
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by surlydev** [comment]
Good afternoon. 

This looks like a fine project, with some wonderfully and well thought out features.

Congrats to anyone who contributed to this project.
